docs/index.md

```diff
@@ -6,12 +6,14 @@ permalink: index.html
 folder: core_lib_doc
 toc: false
 ---
+<p><img src="images/core-lib.png" alt="Company logo" width="240"/></p>
```

Comment:
1. make sure the title "Getting Started" is not there 
2. highlight words like `JSON`, `HTTP`, `NO_CONTENT`, `HTTPStatus.NO_CONTENT`, code fragment like `request_body_dict(request)` 
3. Make sure that the highlight color is as the core-lib color.  +NOT BOLD
4. also the titles color are at the core-lib color